src/protobuf/genProtos.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Can we rename this file to `generate-protos.sh` instead. I'm a bit allergic to camelCase. :)

Reply:
Fixed: bfe97f7